inscriptions/ISic3100.xml

```diff
@@ -177,25 +182,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI331426">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — — —</supplied>ΕΠΑΙΣ
-                    <supplied reason="lost">— — — — — — — — — — — —</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">— — ὑπ(ατίᾳ) τῶν</supplied>
-                    κυρίων ἡμ
-                    <supplied reason="lost">ῶν Ὁνωρίου τὸ ιγʹ</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">καὶ Θεοδο</supplied>σίου τὸ ι
-                    <supplied reason="lost">— — — — — — — — — — — —</supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — — — — — — — — — — — — — — — — — —</supplied>
-                    <lb n="5"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig>ΕΠΑΙΣ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost"><expan><abbr>ὑπ</abbr><ex>ατίᾳ</ex></expan></supplied> <supplied reason="lost">τῶν</supplied> κυρίων ἡμ<supplied reason="lost">ῶν</supplied> <supplied reason="lost">Ὁνωρίου</supplied> <supplied reason="lost">τὸ</supplied> <supplied reason="lost"><num value="13">ιγ</num></supplied>
```

Comment:
The number should be the other way around - num outside, supplied inside. Any information as to how big the supplied extent might be?